index.html

```diff
@@ -1808,78 +1877,87 @@ <h2>Authentication</h2>
+
+      <h3>keyAgreement</h3>
       <p>
-There are at least two suggested methods for implementing Authorization and
-Delegation, which may be layered:
+<code>keyAgreement<code> is a <a>verification relationship</a> which an entity can use to indicate that a <a>verification method</a> is authorized to perform key agreement protocols on behalf of the <a>DID subject</a>. The <em>counterparty(s)</em> in a key agreement protocol can use the <code>keyAgreement</code><a>verification relationship</a> to check whether a party performing a key agreement protocol on behalf of the <a>Did subject</a> is authorized by checking if the <a>verification method</a> used during the keyAgreement protocol is contained in the <code>keyAgreement<code> section of the <a>DID Document</a>.
       </p>
 
-      <ul>
-        <li>
-A <a>verifiable data registry</a> could implement a coarse-grained
-<code>controller</code> pattern by enabling <a>DID documents</a> to express
-the <a>DID</a> of another <a>DID controller</a> that controls it, or
-additionally,
-        </li>
+      <p class="note" title="Uses of keyAgreement">
+If keyAgreement is established, it is up to the verifying party to validate that the <a>verification method</a> used during a keyAgreement protocol is valid and is contained in the <code>keyAgreement</code> section. An example of how this might be used, is when a party wishes to establish a TLS session on behalf of the <a>DID Subject</a> to communicate with another party. In this case, the counterparty would want to check the <a>verification method</a> used during the protocol handshake is contained within the assertionMethod section of the <a>DID Document</a>.
 
-        <li>
-A <a>verifiable data registry</a> could implement a capabilities-based approach enabling
-further fine-grained control of authorization and delegation.
-        </li>
-      </ul>
+      <p>
+A <a>DID document</a> MAY include an <code>keyAgreement</code> property.
+The <code>keyAgreement</code> property is an additional example of a <a>verification relationship</a>.
 
-      <pre class="example nohighlight"
-        title="DID document with a controller property">
+      <p>
+Example:
+      </p>
+
+      <pre class="example nohighlight" title="Authentication field containing three verification methods">
 {
   "@context": "https://www.w3.org/ns/did/v1",
   "id": "did:example:123456789abcdefghi",
-  "controller": "did:example:bcehfew7h32f32h7af3",
-  "service": [{
-    <span class="comment">// used to retrieve Verifiable Credentials associated with the DID</span>
-    "type": "VerifiableCredentialService",
-    "serviceEndpoint": "https://example.com/vc/"
-  }]
-}
-      </pre>
+  <span class="comment">...</span>
+  "keyAgreement": [
+    <span class="comment">// this method can be used to assert statements as did:...fghi</span>
+    "did:example:123456789abcdefghi#keys-1",
+    <span class="comment">// this method can be used to assert statements as did:...fghi</span>
```

Comment:
Need to update the comment text here to refer to performing key agreement protocols as well (not for asserting statements).

```suggestion
    <span class="comment">// this method can be used to perform key agreement
    protocols for did:...fghi</span>
```